cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -71,6 +71,11 @@ public MainServer(ServerConfiguration serverConfiguration) {
 	public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
+
+        if(mods.getAvailableMods().isEmpty()) {
+        	logger.warn("No mods were loaded from the different folders. Please check your installation.");
```

Comment:
yeah I did not copy the right line of code : `logger.info("Loading mods in " + directory.toAbsolutePath());`